cart-page.css

```diff
@@ -0,0 +1,114 @@
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
```

Comment:
Not sure why you need to import same thing twice.

Reply:
I also don't remember why I did that 😅.
I will remove it.